Shared/Samples/Select features in scene layer/SelectFeaturesInSceneLayerView.swift

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SelectFeaturesInSceneLayerView: View {
+    
+    // The point on the screen where the user tapped.
+    //
+    @State private var tapPoint: CGPoint?
```

Comment:
See other samples for the docstring styling. Use `cmd` `option` `/` to let Xcode generate a stub for comment or docstring.